src/CargoBuild/sdk/Sdk.props

```diff
@@ -0,0 +1,94 @@
+
+		<!-- Don't generate editor config file -->
+		<GenerateMSBuildEditorConfigFile Condition="'$(GenerateMSBuildEditorConfigFile)' == ''">false</GenerateMSBuildEditorConfigFile>
+
+		<!-- Don't log the high priority message mentioning this project's name (or copy the product we didn't build). -->
+		<SkipCopyBuildProduct Condition="'$(SkipCopyBuildProduct)' == ''">true</SkipCopyBuildProduct>
+
+		<!-- Don't automatically reference assembly packages since NoTargets don't need reference assemblies -->
+		<AutomaticallyUseReferenceAssemblyPackages Condition="'$(AutomaticallyUseReferenceAssemblyPackages)' == ''">false</AutomaticallyUseReferenceAssemblyPackages>
+		<NoCompilerStandardLib Condition="'$(NoCompilerStandardLib)' == ''">false</NoCompilerStandardLib>
+		<NoStdLib Condition="'$(NoStdLib)' == ''">true</NoStdLib>
+
+		<!-- Disable Visual Studio's Fast Up-to-date Check and rely on MSBuild to determine -->
+		<DisableFastUpToDateCheck Condition="'$(DisableFastUpToDateCheck)' == ''">true</DisableFastUpToDateCheck>
+	</PropertyGroup>
+
+	<ItemDefinitionGroup Condition=" '$(NoTargetsDoNotReferenceOutputAssemblies)' != 'false' ">
+		<ProjectReference>
+			<!--
+        Setting ReferenceOutputAssembly skips target framework cross-project validation in NuGet.  Since NoTargets projects don't define runtime
+        constraints like a target framework, there's no point in checking the compatibilty of project references.
+      -->
+			<ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+			<SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
+		</ProjectReference>
+	</ItemDefinitionGroup>
+
+	<Target Name="CreateManifestResourceNames" />
+
+	<Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" Condition=" '$(MicrosoftCommonPropsHasBeenImported)' != 'true' "/>
+
+	<Import Project="$(CustomAfterNoTargetsProps)" Condition=" '$(CustomAfterNoTargetsProps)' != '' And Exists('$(CustomAfterNoTargetsProps)') " />
+
+	<!-- For CPS/VS support. Importing in .props allows any subsequent targets to redefine this if needed -->
+	<Target Name="CompileDesignTime" />
+
+	<!-- Rust specific props -->
+	<PropertyGroup>
+		<AuthMode Condition="'$(AuthMode)' == '' and '$(Q_SESSION_GUID)' != ''">false</AuthMode>
+		<AuthMode Condition="'$(AuthMode)' == '' and '$(TF_BUILD)' != ''">false</AuthMode>
+		<AuthMode Condition="'$(AuthMode)' == ''">true</AuthMode>
+		<CargoRegistryFeedName Condition="'$(CargoRegistryFeedName)' == ''">cargo.io</CargoRegistryFeedName>
+		<BuildCommandArgs Condition="'$(BuildCommandArgs)' == ''"></BuildCommandArgs>
+		<TestCommandArgs Condition="'$(TestCommandArgs)' == ''"></TestCommandArgs>
+		<CleanCommandArgs Condition="'$(CleanCommandArgs)' == ''"></CleanCommandArgs>
+		<RunCommandArgs Condition="'$(RunCommandArgs)' == ''"></RunCommandArgs>
+		<DocCommandArgs Condition="'$(DocCommandArgs)' == ''"></DocCommandArgs>
+		<!--For internal Microsoft use only, ignore.-->
```

Comment:
Who is the "ignore" directive for? Not sure it makes sense to have this part of the comment.